cmd/trace-agent/subcommands/info/command.go

```diff
@@ -41,15 +44,15 @@ func MakeCommand(globalParamsGetter func() *subcommands.GlobalParams) *cobra.Com
 func runTraceAgentInfoFct(params *subcommands.GlobalParams, fct interface{}) error {
 	return fxutil.OneShot(fct,
 		config.Module(),
-		fx.Supply(coreconfig.NewAgentParams(params.ConfPath, coreconfig.WithFleetPoliciesDirPath(params.FleetPoliciesDirPath))),
+		fx.Supply(coreconfig.NewAgentParams(params.ConfPath, coreconfig.WithConfigName(params.ConfigName), coreconfig.WithFleetPoliciesDirPath(params.FleetPoliciesDirPath))),
```

Comment:
Why add `WithConfigName` here ? Is that required ?

Reply:
No particular reason. I removed it.